lib/Experian/IDAuth.pm

```diff
@@ -309,7 +309,13 @@ sub _get_result_proveid {
 
     return unless $credit_reference and $kyc_summary;
 
-    my $decision = {matches => []};
+    my $decision = {matches => [], kyc_summary_score => 0};
+
+    # calculate kyc summary score
+    if ($kyc_summary->findvalue('FullNameAndAddress/Count') > 0 && $kyc_summary->findvalue('DateOfBirth/Count') > 0)
+    {
+        $decision->{kyc_summary_score} = ($kyc_summary->findvalue('FullNameAndAddress/Count') + $kyc_summary->findvalue('DateOfBirth/Count'));
```

Comment:
IIRC you can make temporary lexicals in the `if` block here to simplify/reduce method calls a bit:

```perl
if ( ( my $fCount = $kyc_summary->findvalue('FullNameAndAddress/Count') ) > 0
    && ( my $dCount = $kyc_summary->findvalue('DateOfBirth/Count') ) > 0 )
{
    $decision->{kyc_summary_score} = $fCount + $dCount;
}
```

Reply:
That'd be great, I'll look into it, thanks!